CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 All notable changes to `Browsershot` will be documented in this file
 
+## 3.34.0 - 2019-11-06
+
+- add 'cookie' (Andreas Schnederle-Wagner, Futureweb OG, schnederle@futureweb.at, #350)
```

Comment:
Please remove your personal data here.

Reply:
According to your Contributing Guidelines: Contributions are welcome and will be fully credited.
So please tell me how to format / word this so won't have another 3-4 changes until it meets your demands ;-)

(btw - "Allow edits from maintainers" is activated)